packages/eslint-plugin/utils.js

```diff
@@ -60,6 +60,51 @@ function isDependencyBlock( jsdoc ) {
 	);
 }
 
+function isImported( node ) {
+	if ( ! node || ! node.type ) {
+		return false;
+	}
+
+	const importTypes = [
+		'ImportDefaultSpecifier',
+		'ImportSpecifier',
+		'ExportDefaultDeclaration',
+		'ExportDeclaration',
+		'ImportDeclaration',
+	];
+
+	if ( importTypes.includes( node.type ) || importTypes.includes( node?.parent?.type ) ) {
```

Comment:
No need to use condition with `node` since this is checked first.
```suggestion
	if ( importTypes.includes( node.type ) || importTypes.includes( node.parent?.type ) ) {
```